app/models/repository.rb

```diff
@@ -21,4 +21,77 @@ def self.clear_all
   def url
     "https://github.com/" + owner + "/" + name
   end
+
+  def release_notes_new_features
+    release_notes = latest_release_notes.downcase
+    release_notes_from_header = release_notes_features_header release_notes
+    unless release_notes_from_header.nil?
+      return release_notes_from_header
+    end
+
+    # if no markdown header was found, attempt to find a line with a colon
+    # and take everything until an empty line is found
+
+    release_notes_from_new_line = release_notes_new_line release_notes
+    unless release_notes_from_new_line.nil?
+      return release_notes_from_new_line
+    end
+
+    latest_release_notes
+  end
+
+  private
+  def release_notes_features_header(release_notes)
+    features_indices = Services::Monitoring::NEW_FEATURES.map { |p| release_notes.index p }.filter{ |p| !p.nil? }.sort
```

Comment:
`filter { |p| !p.nil? }` -> `filter(&:itself)` - `itself` is exactly what you think it is, and it will reject all the nil instances. The other option is `reject(&:nil?)` if you prefer.